test/topology_experimental_raft/test_topology_ops.py

```diff
@@ -49,3 +62,56 @@ async def test_topology_ops(request, manager: ManagerClient):
     logger.info(f"Decommissioning node {servers[0]}")
     await manager.decommission_node(servers[0].server_id)
     await check_token_ring_and_group0_consistency(manager)
+
+    logger.info("Checking results of the background writes")
+    await finish_writes()
+
+    for server in servers:
+        await check_node_log_for_failed_mutations(manager, server)
+
+
+async def check_node_log_for_failed_mutations(manager: ManagerClient, server: ServerInfo):
+    logger.info(f"Checking that node {server} had no failed mutations")
+    log = await manager.server_open_log(server.server_id)
+    occurrences = await log.grep("Failed to apply mutation from")
+    assert len(occurrences) == 0
```

Comment:
This is going to be flaky. Any other reason for failing to apply a mutationand the test will fail.
Such issues are hard to test, I would personally be fine with no test for this bug.

Reply:
> Any other reason for failing to apply a mutationand the test will fail.

There should be no other reasons.

Reply:
It means any time, any other part of the code, fails to apply the mutation, this test will begin failing. This goes against the idea that tests should be owned. Do you want to own a test that someone else can break?

Reply:
Any test can be broken like that due to any random reason due to any random place in the code.

This test is very valuable, it is both a regression test and it allowed us to find out about a different existing problem (fencing during decommission).

Reply:
OK, I don't want to press hard on this. But please keep in mind once you add  a test you marry into it. We don't like removing tests, and going forward try to keep the tests around no matter how bad they are.
Another issue with grepping is that when the error message text changes, and you grep for absence, your test quietly stays passing but broken, and heats the universe.